release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1941,6 +1941,8 @@ controlPlane:
 
 [id="ocp-telco-core-4-17-known-issues_{context}"]
 
+* Setting the `globallyDisableIrqLoadBalancing` field to `false` in the `PerformanceProfile` object reduces the number of CPUs available for IRQ load balancing, as isolated CPUs are excluded. When the `globallyDisableIrqLoadBalancing` field is set to `true`, the isolated CPUs are reflected in `IRQBALANCE_BANNED_CPULIST` instead of `IRQBALANCE_BANNED_CPUS`.(link:https://issues.redhat.com/browse/OCPBUGS-42323[*OCPBUGS-42323*])
```

Comment:
Upgrading and keeping the value intact (either true or false) causes no harm. It is just the true -> false switch performed after upgrading to the affected version that will behave improperly.

Reply:
Hi @MarSik 

How about updating it as follows:

Setting the `globallyDisableIrqLoadBalancing` field from `true` to `false` in the `PerformanceProfile` object reduces the number of CPUs available for rebalancing the load due to isolated CPUs remaining in `IRQBALANCE_BANNED_CPULIST`.

Reply:
due to reserved CPUs remaining..

And the semantics is still slightly wrong. Because the bug does not reduce the number of cpus (the amount was already reduced by the "true" value before). It does not increase the amount as it should.

Reply:
Updated as follows:

* When the `globallyDisableIrqLoadBalancing` field is set to `true` in the `PerformanceProfile` object, the isolated CPUs are reflected in the `IRQBALANCE_BANNED_CPULIST` variable instead of the `IRQBALANCE_BANNED_CPUS` variable. However, setting the `globallyDisableIrqLoadBalancing` field from `true` to `false` does not update the `IRQBALANCE_BANNED_CPULIST` variable correctly. Hence, the number of CPUs available for rebalancing the laod is not increased due to the reserved CPUs remaining in the `IRQBALANCE_BANNED_CPULIST` variable. (link:https://issues.redhat.com/browse/OCPBUGS-42323[*OCPBUGS-42323*])
+
[NOTE]
====
The `IRQBALANCE_BANNED_CPULIST` variable and the `IRQBALANCE_BANNED_CPUS` variable are stored in `/etc/sysconfig/irqbalance`.
====